Tools/scripts/pathfix.py

```diff
@@ -188,15 +196,32 @@ def parse_shebang(shebangline):
     return shebangline[start:]
 
 
+def populate_flags(shebangline):
+    old_flags = b''
+    if keep_flags:
+        old_flags = parse_shebang(shebangline)
+        if old_flags:
+            old_flags = old_flags[2:]
+    if not (old_flags or add_flags):
+        return b''
+    # On Linux, the entire string following the interpreter name
+    # is passed as a single argument to the interpreter.
+    # e.g. #! /usr/bin/python3 -W Error -s == /usr/bin/python3 -W "Error -s"
```

Comment:
hum, no, -W is part of the argument. I suggest:

```suggestion
    # e.g. "#! /usr/bin/python3 -W Error -s" runs "/usr/bin/python3 "-W Error -s"
```